src/highlight/HighlightCanvas.tsx

```diff
@@ -67,6 +67,12 @@ export default class HighlightCanvas extends React.PureComponent<Props> {
 
         shapesArray.forEach(rect => {
             const { height, isActive, isHover, width, x, y } = rect;
+
+            // Ignore empty rects with a width or height of 0
+            if (width === 0 || height === 0) {
```

Comment:
Should we also prevent sending such rects to be stored in the backend?

Reply:
Looking now! Is `combineRects` used to store these rects in the backend?? I see that it's being used in `store/highlight/actions.ts`, although, not sure if that is where we're storing them in the backend...

Reply:
I think I did it! Let me know if I'm missing anything..